app/views/layouts/application.html.erb

```diff
@@ -10,8 +10,12 @@
   </head>
 
   <body>
-    <p class="notice"><%= notice %></p>
+    <% if !notice.nil? %>
```

Comment:
try to use everything inside flash messages but with custom css per message type
{ notice: 'green', warning: 'yellow', danger: 'red', success: 'green'}

Reply:
understood, thanks!